From 2fd4986966373a39334cc83d0346b023bc932166 Mon Sep 17 00:00:00 2001
From: Zheng Xiang <xiang.zheng@linaro.org>
Date: Tue, 22 May 2018 22:19:04 +0800
Subject: [PATCH 1/2] arm64-mmio: support armhf on arm64-mmio machine

Support armhf on arm64-mmio machine so that we can run 32-bit guest on
64-bit host.

Signed-off-by: Zheng Xiang <xiang.zheng@linaro.org>
---
 shared/cfg/machines.cfg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/shared/cfg/machines.cfg b/shared/cfg/machines.cfg
index 250fbdf..f66a591 100644
--- a/shared/cfg/machines.cfg
+++ b/shared/cfg/machines.cfg
@@ -37,7 +37,7 @@ variants:
         del rtc_drift
         del soundcards
     - arm64-mmio:
-        only aarch64
+        only aarch64, armhf
         auto_cpu_model = "no"
         cpu_model = host
         machine_type = arm64-mmio:virt
-- 
1.9.1

